### ISSUE TITLE
Markdown output

### ISSUE THREAD

**Issue by WyriHaximus** [pb33f/openapi-changes#51]
Been using this tool to generate changes for automated PR's like this one: https://github.com/php-api-clients/github-root/pull/438
It doesn't look pretty but it pains a clear enough picture. Would you be open for a PR adding a markdown as an output?

**Comment by daveshanley** [comment]
Oh wow, this is great! I would love to see a markdown output for the summary command. It would be a great upgrade!

**Comment by WyriHaximus** [comment]
Great, let me brush up my Go and see how far I get with a first iteration. No clue how to visualise it yet tho.

**Comment by WyriHaximus** [comment]
Been digging through the source to see how it all works. Don't get everything yet, but I did already start working on possible outputs. These are all based on https://github.com/php-api-clients/github-root/pull/482 as base reference how I'm using this tool now:

### Same as summary but wrapping the tree in ```

Detected Schema changes:

```bash
└─┬Paths
  ├─┬/repos/{owner}/{repo}/actions/secrets/{secret_name}
  │ └─┬PUT
  │   └──[M] description (17035:20)
  ├─┬/orgs/{org}/dependabot/secrets/{secret_name}
  │ └─┬PUT
  │   └──[M] description (9305:20)
  ├─┬/orgs/{org}/actions/secrets/{secret_name}
  │ └─┬PUT
  │   └──[M] description (8734:20)
  ├─┬/repos/{owner}/{repo}/dependabot/secrets/{secret_name}
  │ └─┬PUT
  │   └──[M] description (22969:20)
  └─┬/repositories/{repository_id}/environments/{environment_name}/secrets/{secret_name}
    └─┬PUT
      └──[M] description (33241:20)
```

Date: `08/01/23` | Commit: New: `etc/specs/GitHubEnterprise-3.7/previous.spec.yaml`, Original: `etc/specs/GitHubEnterprise-3.7/current.spec.yaml`
Document Element | Total Changes | Breaking Changes
paths            | 5             | 0               
INFO: Total Changes: 5
INFO: Modifications: 5


### Using markdown to build the tree structure

Detected Schema changes:

* Paths
  * `/repos/{owner}/{repo}/actions/secrets/{secret_name}`
      * `PUT`
        * [`M`] description (17035:20)
  * `/orgs/{org}/dependabot/secrets/{secret_name}`
      * `PUT`
        * [`M`] description (9305:20)
  * `/orgs/{org}/actions/secrets/{secret_name}`
      * `PUT`
        * [`M`] description (8734:20)
  * `/repos/{owner}/{repo}/dependabot/secrets/{secret_name}`
      * `PUT`
        * [`M`] description (22969:20)
  * `/repositories/{repository_id}/environments/{environment_name}/secrets/{secret_name}`
      * `PUT`
        * [`M`] description (33241:20)

Date: `08/01/23` | Commit: New: `etc/specs/GitHubEnterprise-3.7/previous.spec.yaml`, Original: `etc/specs/GitHubEnterprise-3.7/current.spec.yaml`
Document Element | Total Changes | Breaking Changes
paths            | 5             | 0               
INFO: Total Changes: 5
INFO: Modifications: 5

### Using markdown to build the tree structure with emoji

Detected Schema changes:

* Paths
  * `/repos/{owner}/{repo}/actions/secrets/{secret_name}`
      * `PUT`
        * [🔀] description (17035:20)
  * `/orgs/{org}/dependabot/secrets/{secret_name}`
      * `PUT`
        * [🔀] description (9305:20)
  * `/orgs/{org}/actions/secrets/{secret_name}`
      * `PUT`
        * [🔀] description (8734:20)
  * `/repos/{owner}/{repo}/dependabot/secrets/{secret_name}`
      * `PUT`
        * [🔀] description (22969:20)
  * `/repositories/{repository_id}/environments/{environment_name}/secrets/{secret_name}`
      * `PUT`
        * [🔀] description (33241:20)

Date: `08/01/23` | Commit: New: `etc/specs/GitHubEnterprise-3.7/previous.spec.yaml`, Original: `etc/specs/GitHubEnterprise-3.7/current.spec.yaml`
Document Element | Total Changes | Breaking Changes
paths            | 5             | 0               
INFO: Total Changes: 5
INFO: Modifications: 5

**Comment by WyriHaximus** [comment]
@ppmathis I still want to complete this, but honestly, my Go isn't that great, and this project may be more than I can chew on right now. Will have a look at this again this weekend. Been using it a lot on https://github.com/php-api-clients/github-root

**Comment by daveshanley** [comment]
There is a new flag available in `v0.0.56` for the `summary` command. 

`--markdown` or  `-m` will render the tree, tables and summary as markdown and uses emojis. 

Docs have been updated

https://pb33f.io/openapi-changes/summary/
